navigation_and_routing/README.md

```diff
@@ -1,38 +1,38 @@
 # Navigation and Routing sample code
 
 Sample code for [Learning Flutter's New Navigation and Routing System][article],
-an article explaining Navigator 2.0.
+an article explaining the Router widget and it's associated classes.
 
 ## Samples
 
-**Navigator 1.0 samples**
+**Introductory samples**
```

Comment:
Introductory samples sounds a bit weird, Maybe just `Navigator samples`?

Reply:
I think that makes sense, I was concerned that it might be misleading to call these Navigator samples, since `Router` works alongside `Navigator` too

Reply:
I have no strong opinion on this maybe you can add something like` existing navigator sample` or `imperative navigator example` but then you will need to explain what imperative is....

Reply:
Me neither, I'll change this to `navigator`